trino/client.py

```diff
@@ -579,25 +579,25 @@ def process(self, http_response) -> TrinoStatus:
             ):
                 self._client_session.properties.pop(prop, None)
 
-        if constants.HEADER_SET_SESSION in http_response.headers:
+        if http_response.headers.get(constants.HEADER_SET_SESSION):
```

Comment:
I'd instead fix the various `get_session_property_values` and `get_roles_values` etc.

That way you can even unit-test those methods directly instead of using mocks.

It also prevents against regressions when those functions get re-used in other places.

Reply:
Yah, sounds much better to me :)

Change to this approach is in